lib/pf/ipset.pm

```diff
@@ -59,6 +59,10 @@ sub iptables_generate {
     my $cmd = "LANG=C sudo ipset --destroy";
     my @lines = pf_run($cmd);
     my @roles = pf::nodecategory::nodecategory_view_all;
+
+    $cmd = "LANG=C sudo ipset --create portal_deny hash:ip timeout 300 2>&1";
```

Comment:
We can, btw time x is just to create a timeout ipset hash, without the timeout there is no way to change the ipset config.